src/main/java/spoon/support/reflect/CtExtendedModifier.java

```diff
@@ -87,4 +88,14 @@ public int hashCode() {
 		result = 31 * result + (kind != null ? kind.hashCode() : 0);
 		return result;
 	}
+
+	@Override
+	public SourceFragment getOriginalSourceFragment() {
+		return new SourceFragment() {
```

Comment:
I suggest to use same/very similar implementation like `CtElement#getOriginalSourceFragment`. It looks like it should be possible.

Reply:
And then all `getOriginalSourceFragment` may return `ElementSourceFragment`